README.md

```diff
@@ -115,6 +115,10 @@ To start all services, after setup, from the root of this repository, run:
 
     npm run start:services
 
+This will run Docker Compose in "attached" mode. The logs for all containers
+will be output to stdout. To stop all services press <button>Ctrl</button> +
+<button>c</button>.
+
```

Comment:
```suggestion
This will run Docker Compose in "attached" mode. The logs for all containers will be output to `stdout`.  
To stop all services press <kbd>⌃ Control</kbd> + <kbd>C</kbd>.

```

Thanks to: https://kbd.hsuan.xyz/ ;)